tests/integration/high_availability/application_charm/src/continuous_writes.py

```diff
@@ -0,0 +1,65 @@
+# Copyright 2022 Canonical Ltd.
+# See LICENSE file for licensing details.
+
+"""This file is meant to run in the background continuously writing entries to MySQL."""
+
+import sys
+from typing import Dict
+
+import mysql.connector
+from connector import MySQLConnector
+
+
+def continuous_writes(database_config: Dict, table_name: str, starting_number: int) -> None:
+    """Continuously write to the MySQL cluster.
+
+    Args:
+        database_config: a dictionary with MySQL config to connect to the database
+        database_name: the database name where the table continuously written to exists
+        table_name: the table name to direct continuous writes to
+        starting_number: number from which to start writing data to the table (and increment from)
+    """
+    try:
+        with MySQLConnector(database_config) as cursor:
+            cursor.execute(
+                f"CREATE TABLE IF NOT EXISTS `{table_name}`(number INTEGER, PRIMARY KEY(number));"
+            )
+    except Exception:
+        pass
+
+    next_value_to_insert = starting_number
+
+    while True:
+        try:
+            with MySQLConnector(database_config) as cursor:
+                cursor.execute(
+                    f"INSERT INTO `{table_name}`(number) VALUES ({next_value_to_insert})"
+                )
+        except mysql.connector.errors.DatabaseError as e:
+            # errno 2003: can't connect to mysql database
+            if e.errno != 2003:
+                raise
+            continue
+        except Exception as e:
+            if e.errno != 2013 or e.msg != "Lost connection to MySQL server during query":
+                raise
+            continue
```

Comment:
I believe that if this raises an exception, the script will stop. This will cause continuous writes to stop as well, and thus in the tests, we will notice that writes are not increment [here](https://github.com/canonical/mysql-k8s-operator/pull/101/files#diff-b4dad27cf592fea29193749e4ff1dea90ffa41a7949e5b4c3eef94a056528c6aR85-R91)